src/SFML.Graphics/Text.cs

```diff
@@ -298,14 +304,14 @@ public FloatRect GetLocalBounds()
 
         ////////////////////////////////////////////////////////////
         /// <summary>
-        /// Get the global bounding rectangle of the entity.
-        ///
-        /// The returned rectangle is in global coordinates, which means
-        /// that it takes in account the transformations (translation,
-        /// rotation, scale, ...) that are applied to the entity.
-        /// In other words, this function returns the bounds of the
-        /// sprite in the global 2D world's coordinate system.
+        /// Get the GLOBAL bounding rectangle of the text.
         /// </summary>
+        /// <remarks>
+        /// The returned <see cref="FloatRect"/> is in GLOBAL coordinates.
+        /// <para>Transformations (Translation, Rotation, Scale) ARE applied to the entity.</para>
```

Comment:
Same here: `are`

Reply:
Same as GetLocalBounds():
GLOBAL and ARE were capitalized to emphasize the differences between GetLocalBounds() and GetGlobalBounds().  Having emphasis for details like this isn't out of the ordinary from my experience, but if we want to remove that emphasis, I completely understand and can make that change.